crates/iota-sdk/Cargo.toml

```diff
@@ -0,0 +1,41 @@
+[package]
+name = "iota-sdk"
+version = "3.0.0"
+authors = ["IOTA Foundation <info@iota.org>"]
+edition = "2021"
+license = "Apache-2.0"
+description = "The official SDK for IOTA Rebased"
```

Comment:
I think we don't want to mention "Rebased" anymore, it was just used to make the upgrade more clear, but now it should be just `IOTA`